rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -444,11 +469,11 @@ private RMNode getNodebyUrl(String url) {
      * Set a node's state to free, after a completed task by it. Set the to free
      * and move the node to the internal free nodes list. An event informing the
      * node state's change is thrown to RMMonitoring.
-     * 
+     *
      * @param rmNode node to set free.
      * @return true if the node successfully set as free, false if it was down before.
      */
-    private BooleanWrapper internalSetFree(final RMNode rmNode) {
+    BooleanWrapper internalSetFree(final RMNode rmNode) {
```

Comment:
For testing purposes. I have added the annotation `@VisibleForTesting` to make it clear.